sbpy/units/core.py

```diff
@@ -133,3 +136,150 @@ def spectral_density_vega(wfb):
         (flam0.unit, VEGA, lambda x: x / flam0.value,
          lambda x: x * flam0.value)
     ]
+
+
+@u.quantity_input(cross_section='km2', reflectance='1/sr',
+        f_sun=['W/(m2 um)', 'W/(m2 Hz)'])
+def reflectance(cross_section=None, reflectance=None, wfb=None, M_sun=None,
+        f_sun=None):
+    """Reflectance related equivalencies.
+
+    Support conversion from/to reflectance and scattering cross-section
+    to/from total flux and magnitude.
+
+    The default magnitude system is ``VEGAmag``, where the apparent magnitude
+    of Vega is assumed to be 0 in all and any wavelengths and bands.  If other
+    magnitude system is used, then it is implicitly inferred from the solar
+    magnitude passed by keyword 'M_sun'.
+
+    Parameters
+    ----------
+    cross_section : `astropy.units.Qauntity`
+        Total scattering cross-section
+    reflectance : `astropy.units.Quantity`
+        Average reflectance
+    wfb : `astropy.units.Quantity`, `synphot.SpectralElement`, string
+        Wavelength, frequency, or a bandpass of the corresponding flux
+        density being converted.  See
+        :func:`~synphot.SpectralElement.from_filter()` for possible
+        bandpass names.  ``wfb`` overrides ``f_sun`` and ``M_sun`` if
+        present at the same time.
+    f_sun : `astropy.units.Quantity`
+        Solar flux in a unit convertible to the unit of the quantity to be
+        converted.  If ``wfb`` is not provided, then ``f_sun`` will be used in
+        the conversion.  ``f_sun`` overrides ``M_sun`` if both present.
+    M_sun : `astropy.units.Quantity`
+        Solar magnitude in the same magnitude system as the quantity to be
+        converted.  If neither ``wfb`` nor ``f_sun`` is provided, then
+        ``M_sun`` will be used in the conversion.  If ``M_sun`` is not
+        present either, then the default V-band solar magnitude/flux will be
+        used.
+
+    Returns
+    -------
+    eqv : list
+        List of equivalencies
+
+    Examples
+    --------
+    >>> # Convertion between scattering cross-section and reflectance
```

Comment:
Note these examples use built-in constants for V-band.

Reply:
Added.